VERSION

```diff
@@ -1 +1 @@
-1.4.7
+1.4.8
```

Comment:
@ryanhertz, I made sure your PR got a set of eyes on it. I think you're good to merge away, and @stmott can bump his up another version (unless you decide to release together).